i18n/en.json

```diff
@@ -616,7 +620,7 @@
   },
   {
     "id": "api.command_invite.missing_user.app_error",
-    "translation": "Unable to find the user."
+    "translation": "We couldn't find the user. They may have been deactivated by the System Administrator."
```

Comment:
I'm not 100% sure about the message here, maybe something like "We couldn't find the user. The user does not exists or has been deactivated.". I miss there the information about the "not-existence" option.

Reply:
Text is as per spec, so @jasonblais

Reply:
@jespino I'm 0/5. For the general case, "We couldn't find the user." is sufficient. It implies the user may not exist.

The second sentence `They may have been deactivated by the System Administrator.` is added because in a big company you're not always aware if someone leaves. So if `jason` is deactivated, you might do `/invite jason` and be confused why `jason` isn't found when you just chatted with him last week.

So I think it's okay as-is, but if you feel strongly about it, let me know.

Reply:
^The second case is what happened with a customer (though with 5.3 the system message makes it look like they were actually added to the channel, only realizing later on that they're not even in the company anymore)

Reply:
I haven't a strong opinion here, so works form me :)